src/common/transformations/include/transformations/op_conversions/eye_decomposition.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+/**
+ * \ingroup ie_transformation_common_api
+ *
+ * \brief Do eye decomposition to sub-graph (model).
+ */
+class TRANSFORMATIONS_API EyeDecomposition : public MatcherPass {
```

Comment:
It looks like all transformations use the same style in hpp files.
Class Declaration
Description 
Class Definition
e.g. 
![image](https://user-images.githubusercontent.com/21181928/185341510-207fd886-8c1d-4f4c-b2f3-b0915a43dcc1.png)
Please re-write it this way

Reply:
Looks this is the Way, done ;)